test/gtest/common/mem_buffer.cc

```diff
@@ -47,9 +47,10 @@
 bool mem_buffer::is_cuda_supported()
 {
 #if HAVE_CUDA
-    int num_gpus        = 0;
+    const char *file    = "/sys/kernel/mm/memory_peers/nv_mem/version";
+    int num_gpus        = 1;
     cudaError_t cudaErr = cudaGetDeviceCount(&num_gpus);
-    return (cudaErr == cudaSuccess) && (num_gpus > 0);
+    return !access(file, F_OK) && (cudaErr == cudaSuccess) && (num_gpus > 0);
```

Comment:
why need to require nv_peer_mem driver?
ucp should work even w/o it, by pipeline protocols